### PR TITLE
Remove app.flx from apk

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

**Review by mravn-google — APPROVED** [review]
LGTM

File: packages/flutter_tools/gradle/flutter.gradle [hunk 1]
```diff
@@ -438,7 +438,6 @@ class FlutterTask extends BaseFlutterTask {
         return project.copySpec {
             from "${intermediateDir}"
             include "flutter_assets/**" // the working dir and its files
-            include "app.flx"
```
Comment (jakobr-google) [inline packages/flutter_tools/gradle/flutter.gradle]: :1st_place_medal:

**Review by jakobr-google — APPROVED** [review]
LGTM :100: :fireworks:

**Comment by szakarias** [comment]
The red tree is due to flakiness, which has been reported, I'll go ahead and merge this.

**Pull request merged**